src/components/logo/logo.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  Heading,
+  VStack,
+  HStack,
+  Logo,
+  Link,
+  Box,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import appConfig from "appConfig";
+
+const DCLogo = () => {
+  return (
+    <Link
+      isUnderlined={false}
+      aria-label={"Digital Collections Homepage"}
+      href={`${appConfig.DC_URL}`}
+    >
+      <HStack>
+        <Logo
+          name="nyplLionBlack"
+          sizeBasedOn="height"
+          height="40px"
```

Comment:
These are the values in Figma but it looks quite small to me.

Reply:
Same, I'll make sure to check in with Lily at VQA